src/content/structured/components/checkboxes/code.mdx

```diff
@@ -30,21 +30,40 @@ export const snippetsDefault = [
   <ic-checkbox value="extra" label="Extra shot (50p)"></ic-checkbox>
   <ic-checkbox value="Soya milk" label="Soya milk" checked ></ic-checkbox>
   <ic-checkbox value="keep cup" label="Takeaway cup (Reusable keep cups only)" disabled="true" ></ic-checkbox>
-</ic-checkbox-group>`,
+</ic-checkbox-group>
+<script>
+  const checkboxGroup = document.querySelector('ic-checkbox-group');
+  checkboxGroup.addEventListener('icChange', (ev) => console.log(ev.detail.value));
+  const checkboxes = document.querySelector('ic-checkbox');
+  checkboxes.forEach((checkbox) => {
+    checkbox.addEventListener('icCheck', (ev) => {
+      console.log(ev);
```

Comment:
```suggestion
      console.log("onIcCheck", ev);
```
to match preview example

Reply:
I've updated this one to 'icCheck' as it's the web-components snippet.